### DISCUSSION TITLE
Glauber Costa - Turso

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#266] [Announcements]
- https://twitter.com/glcst
- https://twitter.com/tursodatabase
- https://turso.tech/libsql
- https://turso.tech/
- https://github.com/glommer

### libSQL

- [x] What is libSQL and why did you fork sqlite?
- [x] Would you ever consider merging back with sqlite?
- [x] What are some properties of SQlite databases that make them nice to use?
- [x] What are the benefits of running a DB per user?
- [x] What is libSQL-server and how does it relate to libSQL?
- [x] How might someone with an existing sqlite db migrate to libSQL?
- [x] SQLite famously has a ton of tests. What's your strategy for testing?

### Turso

- [x] What is turso and how does it use libSQL?
- [x] How does Turso use the edge?
- [x] What are some challenges around building a distributed database?
- [ ] ~What can you do with extensions?~
- [X] What are embedded replicas?
- [x] How does this approach compare with [Litestream](https://litestream.io/)?
- [x] What do you think about Fly's LiteFS FUSE approach?

### Future

- [ ] What do you think the future of databases is?

**Comment by hipstersmoothie** [comment]
https://rsdoctor.dev
https://www.npmjs.com/package/playwright-test-coverage

**Comment by hipstersmoothie** [comment]
val.town
axo.dev

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/ije/md4w
- https://www.unison.cloud/
